src/components/custom-antd/Form/Form.styled.tsx

```diff
@@ -1,8 +1,18 @@
+import {Form} from 'antd';
+
 import styled from 'styled-components';
 
 import Fonts from '@styles/Fonts';
 
+const {Item: AntdFormItem} = Form;
+
 export const StyledFormItemLabel = styled.label`
   font-family: ${Fonts.nunito};
   font-weight: 300;
 `;
+
+export const StyledFormItem = styled(AntdFormItem)`
+  flex: 1;
+
```

Comment:
Are you sure that it should be applied to all `FormItem`s? Additionally, most likely it means that some existing rules like `width: 100%` should not be needed.

Reply:
I checked and did not find any `FormItem`s with `width: 100%` CSS rule. Did you mean this?

Reply:
I meant `FormItem` items that sometimes were enforcing width, because keeping `FormItem`s width was to small for it, although I may be wrong anyway 👍

Reply:
Again, I did not see `FormItem`s with enforced width.

Reply:
Again, I meant `FormItem` children, not `FormItem` itself :)